Solutions/Menes.Hosting.AspNetCore/Menes/Internal/HttpRequestParameterBuilder.cs

```diff
@@ -422,6 +527,9 @@ private async Task<object> ConvertBodyAsync(OpenApiSchema schema, Stream body)
                 return true;
             }
 
+            //// No default value handling since Path parameters aren't optional if they've been defined (and therefore default values are redundant).
+            //// This is as per the the Open API spec, as seen here: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#parameterObject.
+
```

Comment:
Can the user still specify default value in their OpenAPI specification? If so, is it worth checking at this point to see if there's a default value for the `query` parameter and logging a warning to tell the user they've made a mistake?

Reply:
Nope, I'm convinced.